### PR TITLE
Wip/hughsie/usi global

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/usi-dock/fu-usi-dock-mcu-device.c [hunk 1]
```diff
@@ -729,26 +746,19 @@ fu_usi_dock_mcu_device_cleanup(FuDevice *device,
 {
 	g_autoptr(FwupdRequest) request = fwupd_request_new();
 
+	/* wait for the user to unplug then start the 40 second timer */
+	fu_device_add_private_flag(device, FU_USI_DOCK_DEVICE_FLAG_WAITING_FOR_UNPLUG);
+	fu_device_set_remove_delay(device, 900000);
```
Comment (superm1) [inline plugins/usi-dock/fu-usi-dock-mcu-device.c:751]: The comment says 40s..

Comment (hughsie) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:751]: ohh, it's 40 seconds before we ask the user to plug the cable back in -- but it's much longer before the device comes back. The whole update takes over 15 minutes!

Comment (superm1) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:751]: !!!!

Comment (hughsie) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:751]: Yup, not ideal -- but USI have some ideas on how to speed it up.

**Review by superm1 — APPROVED** [review]

**Pull request merged**